pkg/connector/client.go

```diff
@@ -58,6 +62,20 @@ func NewTwitterClient(login *bridgev2.UserLogin, connector *TwitterConnector, cl
 		userCache: make(map[string]types.User),
 	}
 	client.SetEventHandler(tc.HandleTwitterEvent)
+	tc.matrixParser = &format.HTMLParser{
+		PillConverter: func(displayname, mxid, eventID string, ctx format.Context) string {
+			userID, ok := tc.connector.br.Matrix.ParseGhostMXID(id.UserID(mxid))
+			if !ok {
+				return "@" + displayname
+			}
+			ghost, err := tc.connector.br.GetGhostByID(context.TODO(), userID)
+			if err != nil || len(ghost.Identifiers) < 1 {
+				return "@" + displayname
```

Comment:
Probably better to not assume a specific format on displaynames

```suggestion
				return displayname
			}
			ghost, err := tc.connector.br.GetGhostByID(context.TODO(), userID)
			if err != nil || len(ghost.Identifiers) < 1 {
				return displayname
```